examples/07-EMIT/ComputeChannelSeparation.py

```diff
@@ -0,0 +1,443 @@
+                    or abs(val - range_start) < 1.0
+                    or abs(val - range_stop) < 1.0
+                )
+
+            if (
+                fuzzy_in_range(tx_start, rx_start, rx_stop)
+                or fuzzy_in_range(tx_stop, rx_start, rx_stop)
+                or fuzzy_in_range(rx_start, tx_start, tx_stop)
+                or fuzzy_in_range(rx_stop, tx_start, tx_stop)
+            ):
+                overlapping.append((tx_radio, tx_band))
+    return overlapping
+
+###############################################################################
+# Iterate over all the receivers
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Iterates over each of the receivers in the project and finds any transmit
+# bands that contain overlapping channel frequencies.
+overlapping = []
+for rx_radio in emitapp.results.get_radio_names(modeRx):
+    print("Potential in-band issues for Rx Radio: {}".format(rx_radio))
+    for rx_band in get_rx_bands(rx_radio):
+        tx_bands = overlapping_tx_bands(rx_band)
+        if len(tx_bands) < 1:
+            print('    Rx Band "{}" has no overlapping Tx bands'.format(rx_band[1]))
+            continue
+        print(
+            '    Rx band "{}" has the following overlapping Tx bands:'.format(rx_band[1])
+        )
+        for tx_band in tx_bands:
+            overlapping.append((rx_band, tx_band))
+            print('        {}'.format(tx_band))
+
+###############################################################################
+# Print a list of overlapping bands
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# Prints a list of overlapping receivers and bands
+print(overlapping[0][0])
+
+###############################################################################
+# Analyze the results
+# ~~~~~~~~~~~~~~~~~~~
+# Consider any EMI margin of 0dB or greater an interference issue    
+verbose = False    
+threshold = 0. 
+
+def minimum_tx_channel_separation(rx_band, tx_band, emi_threshold):
+    """Return the minimum separation that the Tx must be operated for interference-free
+    operation of the Rx.
+    Returns:
+        Separation in MHz.
+    Arguments:
+        rx_band - Rx band, given as a tuple: ("Rx Radio Name", "Rx Band Name").
+        tx_band - Tx band, given as a tuple: ("Tx Radio Name", "Tx Band Name").
+        emi_threshold - Tx channel separation to be determined such that the EMI
+        margin is not at or above this level.
+    """
+
+    domain = emitapp.interaction_domain()
+    domain.set_receiver(rx_band[0], rx_band[1], 0.0)
+    radTx = []
+    bandTx = []
+    chanTx = []
+    radTx.append(tx_band[0])
+    bandTx.append(tx_band[1])
+    chanTx.append(0.0)
+    domain.set_interferers(radTx, bandTx, chanTx)
+    
+    interaction = rev.run(domain)
+    worst = interaction.get_worst_instance(modeEmi)
+    # If the worst case for the band-pair is below the EMI limit, then
+    # there are no interference issues and no offset is required.
```

Comment:
In general, you haven't used periods in comments within the code. Here in this line and in line 259 you do. Given the length of the comment, I think it's OK to keep them--but removing them would be more consistent.